lib/wallaby/browser.ex

```diff
@@ -857,6 +857,16 @@ defmodule Wallaby.Browser do
     end
   end
 
+  defp do_get(query, elements) do
+    getNumber = query.conditions[:get]
```

Comment:
as you have the `getNumber` (soon `get_number`) in the query module, I'd prefer to use that so that we can encapsulate that detail there :)